### DISCUSSION TITLE
Push/Pull in VS code

### DISCUSSION THREAD

**Discussion by whereisananya** [giswqs/geog-312#61] [Q&A] [answered]
When I synchronize the changes of GitHub repository in VS code. It prompts me "the action that will pull/push the commit from/to the main or origin". How do I only pull changes from the remote repository instead of pull and push both? 

PS: I don't want to make any changes to the main, only update the changes from the main.

**Comment by mk184** [comment]
![20240918_113816](https://github.com/user-attachments/assets/39fbac89-1787-4078-a3d3-ed88350139d0)
I was facing the same issue

**Comment by giswqs** [comment]
See https://github.com/giswqs/geog-312/discussions/35#discussioncomment-10499070

**Comment by mk184** [comment]
Please I have done so, it is still not working
![20240918_120305](https://github.com/user-attachments/assets/61c62b92-a35b-4972-baea-8aee11cc353f)
![20240918_120234](https://github.com/user-attachments/assets/f74d02e3-a68e-4a60-912a-7049c60a64a3)

**Reply by giswqs** [reply]
There should be no files under the staged changes. Click on the minus sign to unstage them and discard all changes

**Reply by mk184** [reply]
Yes please it worked 

But I can't see any update to the functions classes.

Please I was having the issues I deleted the document "github" from my computer storage

![20240918_122221](https://github.com/user-attachments/assets/31f6f7b4-74ab-48c4-8304-6e1eb4911a64)

**Reply by mk184** [reply]
Please this is my issue now 
![20240918_123154](https://github.com/user-attachments/assets/d119652c-092b-45a6-a2bd-f820cdf7442d)

**Reply by giswqs** [reply, marked as answer]
This is likely your network problem. Try again later.

Alternatively, delete your local repo and clone again

**Comment by mk184** [comment]
Thank you. It worked out
